signature/src/lib.rs

```diff
@@ -13,7 +13,8 @@ extern crate alloc;
 #[cfg(feature = "std")]
 extern crate std;
 #[cfg(feature = "std")]
-use std::{error::Error, vec::Vec};
+use core::error::Error;
+use std::vec::Vec;
```

Comment:
This isn't correct -- the whole point is to only bring in `vec` if the `std` feature is enabled:
```suggestion
#[cfg(feature = "std")]
use std::vec::Vec;
use core::error::Error;
```